src/seedu/addressbook/data/person/Person.java

```diff
@@ -10,10 +10,13 @@
  */
 public class Person implements ReadOnlyPerson {
 
+    static private int nextSequenceNumber = 0;
```

Comment:
recollect the tutorial discussion why assigning this to 1 makes more sense than assigning to 0